packages/client/components/ScopePhaseArea.tsx

```diff
@@ -143,7 +145,7 @@ const ScopePhaseArea = (props: Props) => {
 
   return (
     <ScopingArea isDesktop={isDesktop}>
-      <StyledTabsBar activeIdx={activeIdx}>
+      <StyledTabsBar className='scope-phase-area' activeIdx={activeIdx}>
```

Comment:
-1 instead of creating a global style, these settings should be added in the `StyledTabsBar` line 43-48

Reply:
Hello @Dschoordsch, suggested changes have been performed you can take another look.